tools/developers/launch_instance.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${AWS_USER}" ]; then
+    echo "Please set the AWS_USER environment variable"
+    exit 1
+fi
+
+AWS_SSH_KEYNAME="${AWS_USER}.ssh"
+AWS_REGION="us-east-2"
```

Comment:
Aha now I understand that this is intended, but it would be nice if we could get instances in other regions as well.